src/Enghouse/readme.md

```diff
@@ -0,0 +1,89 @@
+
+![EI **CCaaS** Watson Asistance Integration Architecture Overview](ibmWatson.jpg)
+
+The client-side component manages the communication between the user and the agent. It implements the service desk API that is fully supported by the Watson Assistant web chat integration. (For more information about this API, see [ServiceDesk API](https://github.com/watson-developer-cloud/assistant-web-chat-service-desk-starter/blob/main/docs/API.md)).
+
+The communication uses the EI CustomChat widget component.  This can be referenced using the EI **CCaaS** default page or the [hosting page](./client/indexWatson.html) can be managed by the customer - simply add a script tag to load the EnghouseCCaaSWatsonAssistance.js from the EI **CCaaS** servers and pass the parameters to your page in the URL as detailed below. A [hosting page](./client/indexWatsonOptions.html) may also set the `watsonAssistantChatOptions` prior to loading the EnghouseCCaaSWatsonAssistance.js for more advanced features, or to set the integration parameters in HTML rather than via URL (if both set URL will override HTML), e.g.:
+
+```
+<script>
+ configuration
+  window.watsonAssistantChatOptions = {
+    integrationID: '[integrationID]',
+    region: '[region]',
+    serviceInstanceID: '[serviceInstanceID]',
+    subscriptionID: null,
+    debug: true,
+    carbonTheme: 'g100'
+  };
+
+</script>	
+<script src="https://ccsp.ei.eilabonline.biz/PS.CustomChat/EnghouseCCaaSWatsonAssistance.js"></script>
+```
+
+
+## Setting Up
+
+
+The integration utilises the flexible Custom Chat integration to EI **CCaaS**, the steps for deployment are:
+
+
+1. Configure Chat routing in EI **CCaaS** tenant with desired Queue and Skills.
+2. Configure CustomChat "widget" and copy its URL used to control the Live Agent interaction specific to your needs, and a "WA" URL to access the WA integration from EI **CCaaS**.
+3. When using the integration, the "widget" URL is required to be passed either as the *ccURL* parameter to the "WA" URL, or as a variable in the WA Dialog called *CCURL*.  The "WA" URL must also include parameters to utilise your Assistant's *integrationID*, *region*, and *serviceInstanceID*.
+
+
+
+## Example
+
+If the CustomChat widget" URL was:
+
+`https://{Enghouse CCaaS FQDN}/SocialConnectorHelperAPI/api/customchat/getWidgetConfiguration?WidgetId=<*widgetID*>`
+
+...you would set this to the variable *CCURL* in the Dialog, prior to any likelihood of needing to speak to a Live Agent.
+
+
+If the Assistant URL provided to you by Enghouse Interactive was:
+
+`https://{Enghouse CCaaS FQDN}/PS.CustomChat`
+
+...you would access your Assistant dialog pass your Assistants' parameters to it (using default EI **CCaaS** Watson page in this example):
+
+`https://{Enghouse CCaaS FQDN}/PS.CustomChat/indexWatson.html?integrationID=<*integration-ID*>&region=<*region*>&serviceInstanceID=<*service-ID*>`
```

Comment:
I think I would remove any references to `indexWatson.html` in here. I don't see that customers are going to want to do this. By using your HTML file all the customer will get is an empty webpage with web chat on it but they are going to want to embed web chat on their own webpage using their own HTML file. So not sure I see much value in having these HTML files mentioned in this readme or even included in the repo. It would be good to include an example HTML file that is like what a customers HTML file would look like but such a file would not include any URL parameters like you are using here. I will include such an example HTML file in my email to you guys. You can either check that HTML file into the repo or just put the contents of the HTML file here in the readme.

Reply:
indexWatson.html is just example , We explained it in our readme file as below:
This can be referenced using the EI CCaaS default page or the [hosting page](https://github.com/Enghouse-CCSP-PS/assistant-web-chat-service-desk-starter/blob/main/src/Enghouse/client/indexWatson.html)